src/request/import/ImportWords.js

```diff
@@ -84,11 +83,12 @@ class ImportWords {
 
         this._passwordSetter.on(PassphraseSetterBox.Events.ENTERED, () => {
             let colorClass = '';
-            if (this._keyResults.entropy.addresses) {
+            if (this._secrets.entropy) {
+                const key = new Key(this._secrets.entropy, false);
                 const color = Iqons.getBackgroundColorIndex(
                     new Nimiq.Address(
                         // use color of first address as loginFile color
-                        this._keyResults.entropy.addresses[0].address,
+                        key.deriveAddress('m/44\'/242\'/0\'/0\'').serialize(),
```

Comment:
use Contants.DEFAULT_KEY_PATH :)

Reply:
Is already in the Export PR. Wanted to change it there once merged, but will do here as well now.